redis/tests/support/mod.rs

```diff
@@ -205,6 +205,13 @@ impl RedisServer {
         &self.addr
     }
 
+    pub fn get_conn_info(&self) -> redis::ConnectionInfo {
```

Comment:
I think we should drop the `get_()` prefix (per the API guidelines). For a public method I would probably expand `conn` to `connection`, in line with the type name.